spec/controllers/submissions_controller_spec.rb

```diff
@@ -854,13 +854,48 @@
       end
 
       describe '#manually_collect_and_begin_grading' do
-        before do
+        it('it should respond with 302') do
           post_as grader, :manually_collect_and_begin_grading,
                   params: { course_id: course.id, assignment_id: @assignment.id, grouping_id: @grouping.id,
                             current_revision_identifier: revision_identifier }
+
+          expect(response).to have_http_status :found
         end
 
-        it('should respond with 302') { expect(response).to have_http_status :found }
+        it 'should not flash any error messages' do
+          expect_any_instance_of(SubmissionsController).not_to receive(:flash_message)
```

Comment:
Because there are many ways of setting the `flash` beyond just calling `flash_message`, prefer instead checking the response's flash value and asserting that it's empty.